src/resources/Dissolutions/soloPropCompany.ts

```diff
@@ -0,0 +1,141 @@
+import { DissolutionResourceIF } from '@/interfaces'
+import { BulletListTypes, CorpTypeCd, FilingCodes, ItemTypes, RouteNames, ViewComponentNames } from '@/enums'
+import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
+import { CorpEntityDissolutionSteps } from '@/resources/Dissolutions/stepTemplates'
+
+// SB TODO : cleanup this code
+export const soloPropCompanyDissolutionResource: DissolutionResourceIF = {
+  entityType: CorpTypeCd.SOLE_PROP,
+  displayName: GetCorpFullDescription(CorpTypeCd.BENEFIT_COMPANY),
+  steps: CorpEntityDissolutionSteps,
+  filingData: [{
+    entityType: CorpTypeCd.BENEFIT_COMPANY,
+    filingTypeCode: FilingCodes.DISSOLUTION_VOLUNTARY
+  }],
+  detailsTitle: 'Company Details',
+
+  affidavit: {
```

Comment:
All these are needed to component work even we are not explicitly using it. When I am removing these its giving error.

Reply:
Which components are giving you an error? They will need to be fixed to allow for this data to not exist. Having dummy data here is not a good solution. (It looks like it does something, but it actually does nothing, leading to confusion and increased code maintenance cost.)

Reply:
Is it the components giving errors or are they type errors from not adhering to the interface?

For example , `steps` i know is optional in your flow, so it might make sense to create an interface unique to Firms or make this property optional in the IF. 

If the process is failing, it's certainly worth looking into, as a lot of the current components rely on this templated data. |
If there is no text content difference between firms and corporations, then this resourcing could potentially feel redundant, but we've found this easier to make changes after the fact :)

But yes, you might not require all of this, and if so, please remove and allow optional values in the IF or perhaps a unique firm dissolutionIF.

Reply:
, I was getting error when I remove ```reviewAndConfirm``` section , actually, there is one section  ```completingPartyStatement``` was used inside Certify component

Reply:
This resources file looks good now. (Only contains the objects it needs 👍 )

Reply:
@cameron-freshworks Can we resolve this conversation?

Reply:
@severinbeauvais You bet!